### ISSUE TITLE
Add dialog icon not responsive for mobile view

### ISSUE THREAD

**Issue by naina-verma** [almighty/almighty-ui#545]
Add dialog icon not responsive for mobile view
![screenshot from 2016-12-20 11-51-19](https://cloud.githubusercontent.com/assets/19172870/21340171/c40ce202-c6aa-11e6-950c-1ce37702f4b7.png)

**Comment by SMahil** [comment]
@Mgranfie @mindreeper2420 In mobile view can you share the position for Add WorkItem Button

**Comment by mindreeper2420** [comment]
@SMahil @Mgranfie the position should not differ from the desktop view - it should stay on the right side, overlaying the menu and the list. Similar to the tooltip question, think of it as a FAB from Material Design. It is a fixed position element.

**Comment by SMahil** [comment]
@mindreeper2420 i guess we need to make header responsive for this.... Its not button which is breaking.. its header which is breaking on mobile and desktop view

**Comment by mindreeper2420** [comment]
@SMahil yes, I would say that the header is definitely broken, going off of the screenshot. Fixing that to be a hamburger menu (which is built into PatternFly) and then have the FAB in the upper right area would fix the alignments and give the list more room.

**Comment by mindreeper2420** [comment]
@SMahil Here is the horizontal navigation that utilizes the responsive methodology: http://www.patternfly.org/pattern-library/navigation/horizontal-navigation/#_code. In the code, there is a `<button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse-1">` that should help to remedy the navigation issue.

**Comment by SMahil** [comment]
For Header Responsive behaviour I have raised the PR : #597
